CHANGELOG.md

```diff
@@ -3,10 +3,10 @@
 ## Version 0.4.1-dev (changes since 0.4.1 go here)
 
 ### Features
-* Added `skbio.io.format.stockholm` for reading Stockholm files into a `TabularMSA`. ([#967](https://github.com/biocore/scikit-bio/issues/967))
+* Added `skbio.tree._tree.bifurcate` for converting multifurcating trees into bifurcating trees.* Added `skbio.io.format.stockholm` for reading Stockholm files into a `TabularMSA`. ([#967](https://github.com/biocore/scikit-bio/issues/967))
```

Comment:
change `skbio.tree._tree.bifurcate` to `skbio.tree.TreeNode.bifurcate`

add https://github.com/biocore/scikit-bio/issues/896 markdown link to end of line (similar to other changelog entries)